addons/test/misra/misra-test.c

```diff
@@ -233,6 +233,17 @@ void misra_7_3() {
   long double misra_7_3_e = 7.3l; //7.3
   }
 
+void misra_7_4_const_call(int a, const char* b) { }
+void misra_7_4_call(int a, char* b) { }
```

Comment:
It seems that travis is [worry](https://travis-ci.org/github/danmar/cppcheck/jobs/739901221#L2485) about unused arguments.

```suggestion
void misra_7_4_const_call(int a, const char* b) { } // 2.7
void misra_7_4_call(int a, char* b) { } // 2.7
```

Reply:
Thanks, I have fixed this now.